### PR TITLE
fix: Optimize the display of elements of the block type.

### PR DESCRIPTION
**Pull request by chenshuai2144**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog                                                              |
| ---------- | ---------------------------------------------------------------------- |
| 🇺🇸 English | Optimize the display effect of Descriptions when there is more content |
| 🇨🇳 Chinese | 优化 Descriptions 在内容比较多时的显示效果                             |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

[View rendered components/date-picker/index.en-US.md](https://github.com/ant-design/ant-design/blob/fix-descriptions/components/date-picker/index.en-US.md)
[View rendered components/descriptions/demo/text.md](https://github.com/ant-design/ant-design/blob/fix-descriptions/components/descriptions/demo/text.md)
[View rendered components/table/demo/custom-filter-panel.md](https://github.com/ant-design/ant-design/blob/fix-descriptions/components/table/demo/custom-filter-panel.md)
[View rendered docs/react/customize-theme.zh-CN.md](https://github.com/ant-design/ant-design/blob/fix-descriptions/docs/react/customize-theme.zh-CN.md)
[View rendered docs/react/faq.en-US.md](https://github.com/ant-design/ant-design/blob/fix-descriptions/docs/react/faq.en-US.md)

### CODE REVIEW

File: components/descriptions/Cell.tsx [hunk 1]
```diff
@@ -54,16 +54,31 @@ const Cell: React.FC<CellProps> = ({
...
-        >
-          {label}
-        </span>
-      )}
-      {content && <span className={classNames(`${itemPrefixCls}-item-content`)}>{content}</span>}
+      <div
+        style={{
+          display: 'flex',
+        }}
+      >
```
Comment (afc163) [inline components/descriptions/Cell.tsx:61]: 加个 className 吧。

**Review by xrkffgg — APPROVED** [review]

File: components/descriptions/Cell.tsx [hunk 2]
```diff
@@ -54,16 +54,27 @@ const Cell: React.FC<CellProps> = ({
...
+              [`${itemPrefixCls}-item-no-colon`]: !colon,
+            })}
+          >
+            {label}
+          </span>
+        )}
+        {content && (
+          <span
+            style={{
+              flex: 1,
```
Comment (zombieJ) [inline components/descriptions/Cell.tsx:70]: why inline?

**Review by zombieJ — APPROVED** [review]

**Comment by zombieJ** [comment]
changelog 再改改，这个别人看不懂

File: components/descriptions/Cell.tsx [hunk 3]
```diff
@@ -54,16 +54,18 @@ const Cell: React.FC<CellProps> = ({
...
-        <span
-          className={classNames(`${itemPrefixCls}-item-label`, {
-            [`${itemPrefixCls}-item-no-colon`]: !colon,
-          })}
-        >
-          {label}
-        </span>
-      )}
-      {content && <span className={classNames(`${itemPrefixCls}-item-content`)}>{content}</span>}
+      <div className={`${itemPrefixCls}-item-container`}>
```
Comment (paranoidjk) [inline components/descriptions/Cell.tsx:57]: - 这个 dom 结构的改动导致 https://github.com/ant-design/ant-design/blob/master/components/descriptions/style/index.less#L97 这个样式选择器失效了
- 从而导致用户自定义 label 可能会样式错乱 https://riddle.alibaba-inc.com/riddles/28aa6173

Comment (yoyo837) [reply components/descriptions/Cell.tsx:57]: 搞个PR改成 `${itemPrefixCls}-item-container`

Comment (paranoidjk) [reply components/descriptions/Cell.tsx:57]: @yoyo837 这是不够的，多了一层 div 结构

Comment (yoyo837) [reply components/descriptions/Cell.tsx:57]: 直接PR改改吧，内部链接看不了

Comment (yoyo837) [reply components/descriptions/Cell.tsx:57]: 直接>span挪到container就可以了不？

Comment (zombieJ) [reply components/descriptions/Cell.tsx:57]: 直接来个：
```less
.ant-descriptions-item-label {
  display: flex;
  flex-wrap: nowrap;
}
```

Comment (zombieJ) [reply components/descriptions/Cell.tsx:57]: 我来 patch 一下

**Pull request merged**